### PR TITLE
feature to allot courses for final registration

### PR DESCRIPTION
**Pull request by arrnavvv**

### CODE REVIEW

**Comment by arrnavvv** [comment]
Solved issue #805

File: FusionIIIT/templates/academic_procedures/allot_courses.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            </select>
+        </div>
+
+        <div class="field">
+            <label>Semester</label>
+            <select class="ui dropdown" required="true" name="semester">
+                <option value="NULL" class="sem">Select Semester</option>
+                <option value="1" class="sem">1</option>
+                <option value="2" class="sem">2</option>
+                <option value="3" class="sem">2</option>
```
Comment (tushhr) [inline FusionIIIT/templates/academic_procedures/allot_courses.html]: Typo error...

File: FusionIIIT/templates/academic_procedures/allot_courses.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+<div class="ui active tab segment" data-tab="allotment">
+
+    <form class="ui form" method="POST" action='/academic-procedures/acad_person/allot_courses/'
+        enctype="multipart/form-data" id='allot_courses_form'>
+        {% csrf_token %}
+        <div class="field">
+            <h4 class="ui blue header">Note : Provide the data in Excel Sheet in following format:</h4>
+            RollNo | CourseSlot Name | CourseCode of alloted course
+            <h4 class="ui dividing header"><a href="../media/Administrator/academic_information/sheet.xlsx">Download</a>
```
Comment (tushhr) [inline FusionIIIT/templates/academic_procedures/allot_courses.html]: I don't know, how the "Download" link will work, you haven't pushed anything...

**Review by tushhr — CHANGES_REQUESTED** [review]
@arrnavvv.

**Comment by arrnavvv** [comment]
Solved issue #842

File: FusionIIIT/templates/academic_procedures/allot_courses.html (hunk 2 above)
Comment (arrnavvv) [reply FusionIIIT/templates/academic_procedures/allot_courses.html]: Media folder is gitignored so I cannot add excel in that directory, but it is working.

File: FusionIIIT/templates/academic_procedures/allot_courses.html (hunk 1 above)
Comment (arrnavvv) [reply FusionIIIT/templates/academic_procedures/allot_courses.html]: fixed it

**Comment by MananJethwani** [comment]
@tushhr kindly review this PR again

**Pull request merged**